spec/unit/version_spec.rb

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/version'
+
+describe 'GoodData::Versions' do
```

Comment:
The double colon notation normally signifies a module but `GoodData::Versions` doesn't exist.